src/components/contextpopup/partials/contextpopup.html

```diff
@@ -10,28 +10,28 @@
       <table>
           <tbody>
           <tr>
-              <td>CH1903 / LV03</td>
+              <td><a href="{{'contextpopup_lv03_url' | translate}}" target="_blank">CH1903 / LV03</a></td>
```

Comment:
can you use `translate-attr="{href: 'contextpopup_lv03_url'}" ` the use of `translate` filter is only recommended when we can't use `translate-attr`  because it's interpreted in every digest cycle.